### ISSUE TITLE
can you upload the code of the new art entangled one you uploaded 

### ISSUE THREAD

**Issue by anmayraj** [bgstaal/multipleWindow3dScene#10]

**Comment by JMcrafter26** [comment]
Yes, @bgstaal please upload the new files too, they are awesome!!

**Comment by Slider-Override** [comment]
Ye3ah please please upload the fxhash: https://twitter.com/i/status/1727322594570027343

**Comment by Cryptodemon99** [comment]
Wow

**Comment by jiangjiang2754** [comment]
CSS方式

css部分

<style type="text/css">

    .logo-self {

        font-size: 40px;

        text-shadow: 5px 5px 5px black, 0px 0px 2px black; 

        color: white;

    }

  </style>

页面引用部分

<div class="logo-self">HTML5</div>

2，canvas方式

脚本部分

<script type="text/javascript">

    function drawText() {

        var canvas = document.getElementById("canvas");

        var ctx = canvas.getContext("2d");

        ctx.font = "35px 微软雅黑";

        ctx.strokeText("VVHi",50,50);

    }

  </script>

页面引用部分

<body onload="drawText();">

  <div>

    <canvas id="canvas" width="200" height="100"></canvas>

  </div>

</body>

**Comment by maverickOG** [comment]
yes, could you please share the 'quantum entanglement' code with us? @bgstaal

**Comment by Lobohombo** [comment]
En el idioma de México

Único dueñoDarioRodolfo

**Comment by Cryptodemon99** [comment]
Code fantastic
